package/MDAnalysis/analysis/base.py

```diff
@@ -89,25 +90,28 @@ def _conclude(self):
 
     """
 
-    def __init__(self, trajectory, start=None,
-                 stop=None, step=None, verbose=None, quiet=None):
+    def __init__(self, trajectory, verbose=False, **kwargs):
         """
         Parameters
         ----------
         trajectory : mda.Reader
             A trajectory Reader
-        start : int, optional
-            start frame of analysis
-        stop : int, optional
-            stop frame of analysis
-        step : int, optional
-            number of frames to skip between each analysed frame
-        verbose : bool, optional
-            Turn on verbosity
         """
-        self._verbose = _set_verbose(verbose, quiet, default=False)
-        self._quiet = not self._verbose
-        self._setup_frames(trajectory, start, stop, step)
+        self._trajectory = trajectory
+        self._verbose = verbose
+        # do deprecated kwargs
+        # remove in 1.0
+        deps = []
+        for arg in ['start', 'stop', 'step']:
+            if arg in kwargs and not kwargs[arg] is None:
+                deps.append(arg)
+                setattr(self, arg, kwargs[arg])
+        if deps:
+            raise ValueError(deps)
```

Comment:
I've made this a VE so that I can find all deprecated usages in the testsuite, will remove later..